test/04_build_skip/environment.json

```diff
@@ -0,0 +1,4 @@
+{
+    "CIBW_BUILD": "cp3?-*",
+    "CIBW_SKIP": "cp33-*"
```

Comment:
Python 3.3 is about to be removed from cibuildwheel - maybe better to change this to Python 3.4

Reply:
Changed the folder name anyway, and added the line. So to almost any diff, this file is (almost) completely changed anyway, so might as well clean this up? :-/